cmd/antrea-agent/options.go

```diff
@@ -246,6 +247,9 @@ func (o *Options) validateFlowExporterConfig() error {
 				o.activeFlowTimeout = o.pollInterval
 				klog.Warningf("ActiveFlowExportTimeout must be greater than or equal to FlowPollInterval")
 			}
+			if o.activeFlowTimeout >= connections.StaleConnectionTimeout {
+				return fmt.Errorf("ActiveFlowExportTimeout is greater than or equal to the stale connection timeout(flows are not exported properly)")
+			}
```

Comment:
rather than do this, should we set `StaleConnectionTimeout` to `max(5min, 2*(active timeout), 2*(idle timeout))`. Or do you think that 5 minute is large enough that it's very unlikely that a user would try to use timeouts which are larger?

Reply:
Good idea. I prefer this rather than having a strong gating requirement for active and idel timeouts.